packages/react-instantsearch-core-next/src/useIndex.ts

```diff
@@ -0,0 +1,23 @@
+import index from 'instantsearch.js/es/widgets/index/index';
+import { useEffect, useMemo } from 'react';
+
+import { useIndexContext } from './useIndexContext';
+
+import type { IndexWidgetParams } from 'instantsearch.js/es/widgets/index/index';
```

Comment:
should the imports be grouped somehow? is there good eslint rules for it (if I remember it correctly recently there were some added in the algolia config)

Reply:
They're already grouped following our ESLint rule. I'll revisit the config though because it's weird here indeed.